README.rst

```diff
@@ -303,6 +303,14 @@ This is the list of operators that can be used:
 - ``ilike``
 - ``in``
 - ``not_in``
+- ``any``
+- ``not_all``
+
+any / not_all
+^^^^^^^^^^^^^
+
+PostgreSQL specific operators can be used to filter queries based on presence
+(``any``) or absence (``not_all``) value in model attribute of ARRAY type.
```

Comment:
Does `not_all` describe the absence of a value or that not all the values are equal to the one we're filtering by?

I have suggested the addition of another test (in a different comment) to check for this case.

Reply:
enhanced testcases from 0318f4d revealed different behavior between sqla1.0 and sqla1.1+, more specifically a behavior of `~` / `not_` operator. The implementation was completely changed to support all sqla APIs correctly.